Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/container/WebAppOnLinuxDeployPresenter.java

```diff
@@ -152,7 +153,9 @@ public void onLoadPricingTierList() {
      */
     public void onLoadAppServicePlan(String sid, String rg) {
         Observable.fromCallable(() -> AzureWebAppMvpModel.getInstance()
-                .listAppServicePlanBySubscriptionIdAndResourceGroupName(sid, rg))
+                .listAppServicePlanBySubscriptionIdAndResourceGroupName(sid, rg).stream()
```

Comment:
also remove the filter in view of intelliJ

Reply:
agree